### DISCUSSION TITLE
websocket: bad handshake

### DISCUSSION THREAD

**Discussion by NatsumiXD** [nonebot/discussions#105] [问答]
**描述问题：**

websocket: bad handshake

**如何复现？**

正常链接nonebot2 和 cqhttp

**期望的结果**

握手成功

**环境信息：**

 - OS: Windows
 - Python Version: 3.10.1
 - Nonebot Version: 2.0

**协议端信息：**

 - 协议端: go-cqhttp
 - 协议端版本: v1.0.0-rc1

**截图或日志**

![QQ截图20220218162025](https://user-images.githubusercontent.com/63963655/154644955-3f93c17c-2332-48ee-9885-b41ad4828090.png)
![QQ截图20220218162039](https://user-images.githubusercontent.com/63963655/154644994-00374197-b3cb-4e32-b9df-0764572be7a1.png)

**Comment by mnixry** [comment]
由于NoneBot2版本更新, 现在websocket上报地址已经不再是 `/cqhttp/ws`
请参考 #74 将您的Bot迁移至 NoneBot 2.0.0 beta版本
